docker-compose-cloud.buildx.yaml

```diff
@@ -100,3 +100,19 @@ services:
         platforms:
           - linux/amd64
           - linux/arm64
+  proxy:
```

Comment:
this is for the publish to Cloud action, I don't think we need the proxy for Cloud so we can remove this.

Reply:
Ah ha! I thought this was how we published the containers in general - looks like that's publish_docker.sh